project_completion_report/report/project_completion_report.py

```diff
@@ -2,85 +2,102 @@
-        'hours': fields.float(
-            'Time spent', digits=(16, 2), readonly=True,
-            help="Time spent on timesheet"),
-        'user_id': fields.many2one('res.users', 'User', readonly=True),
-        'project_id': fields.many2one(
-            'project.project', 'Project', readonly=True),
-        'project_state': fields.char(
-            'State', readonly=True, help="Project State"),
-        'activity_stage_id': fields.many2one(
-            'project.task.type', 'Stage',
-            readonly=True, help="Activity Stage"),
-        'account_id': fields.many2one(
-            'account.analytic.account', 'Analytic account', readonly=True),
-        'activity_id': fields.char(
-            'Activity id', readonly=True, help="Task id or Issue id"),
-        'activity_name': fields.char(
-            'Activity name', readonly=True, help="Task name or Issue name"),
-        'planned_hours': fields.float(
-            'Init. time', digits=(16, 2), readonly=True, help="Initial time"),
-        'remaining_hours': fields.float(
-            'Remain. time', digits=(16, 2), readonly=True,
-            help="Remaining time"),
-        'br_id': fields.many2one(
-            'business.requirement', 'Bus. requ.',
-            readonly=True, help="Business requirement"),
-        'partner_id': fields.many2one(
-            'res.partner', 'Customer', readonly=True),
-        'project_categ_id': fields.many2one(
-            'project.project.category',
-            'Project Cat.', readonly=True, help="Project Category"),
-    }
+    id = fields.Integer('ID', readonly=True)
+    partner_id = fields.Many2one(
+        'res.partner', 'Customer', readonly=True)
+    br_id = fields.Many2one(
+        'business.requirement', 'Bus. Req.',
+        readonly=True, help="Business Requirement")
+    project_id = fields.Many2one(
+        'project.project', 'Project', readonly=True)
+    account_id = fields.Many2one(
+        'account.analytic.account', 'Analytic Account', readonly=True)
+    project_categ_id = fields.Many2one(
+        'project.project.category',
+        'Project Cat.', readonly=True, help="Project Category")
+    activity_type = fields.Selection(
+        [
+            ('task', 'Task'),
+            ('issue', 'Issue'),
+        ], 'Type', readonly=True,
+        help="Type is used to separate Tasks and Issues")
+    activity_id = fields.Char(
+        'Activity ID', readonly=True, help="Task ID or Issue ID")
+    activity_name = fields.Char(
+        'Activity Name', readonly=True, help="Task name or Issue name")
+    user_id = fields.Many2one(
+        'res.users',
+        'Assignee',
+        readonly=True,
+        help="Assignee is not necessarily the one who input the Timesheets")
+    activity_stage_id = fields.Many2one(
+        'project.task.type', 'Stage',
+        readonly=True, help="Activity Stage")
+    # FIXME if BR resource UoM is not hours, `qty` needs to be converted

```

Comment:
Can you take care of this too?

Reply:
Do I need to convert the **Day(s)** uom only or should convert **Unit(s)** as well into **Hour(s)**?
@elicoidal @seb-elico

Reply:
@elicoidal it's a tricky one, better to not fix it for now since it might have a huge impact on the performance

@YogeshMahera-SerpentCS please leave the FIXME, we'll fix it later

Reply:
Sure @seb-elico